src/axom/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ axom_add_component(COMPONENT_NAME sina   DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONE
 axom_add_component(COMPONENT_NAME slam   DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONENTS})
 axom_add_component(COMPONENT_NAME primal DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONENTS})
 axom_add_component(COMPONENT_NAME sidre  DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONENTS})
+axom_add_component(COMPONENT_NAME bump   DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONENTS})
```

Comment:
@LLNL/axom General question independent of this PR: is the ordering of `axom_add_component` calls intended to represent strict dependencies among Axom components? If so, then I think this needs some re-assessment.

Reply:
That's what it looked like to me. Would alphabetical be better?